autotest/ogr/ogr_gml_citygml_read.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env pytest
```

Comment:
It would be good also to add a new test case in autotest/osr/osr_epsg.py that just tests the behavior of EPSGTreatsAsLatLong() on EPSG:6697

Reply:
Sure. I added.